static/app/views/starfish/modules/databaseModule/queries.ts

```diff
@@ -472,50 +475,35 @@ export const useQueryMainTable = (options: {
   });
 };
 
+type QueryTransactionByTPMAndP75ReturnType = {
+  count: number;
+  interval: string;
+  p75: number;
+  transaction: string;
+}[];
 export const useQueryTransactionByTPMAndP75 = (
   transactionNames: string[]
-): DefinedUseQueryResult<
-  {count: number; interval: string; p75: number; transaction: string}[]
-> => {
-  const pageFilter = usePageFilters();
-  const {startTime, endTime} = getDateFilters(pageFilter);
-  const dateFilters = getDateQueryFilter(startTime, endTime);
-  const queryFilter = `transaction IN ('${transactionNames.join(`', '`)}')`;
-
-  const query = `
-  select
-    count(DISTINCT transaction_id) as count,
-    floor(quantile(0.75)(exclusive_time), 5) as p75,
-    transaction,
-    toStartOfInterval(start_timestamp, INTERVAL ${INTERVAL} hour) as interval
-  FROM default.spans_experimental_starfish
-  where
-    ${DEFAULT_WHERE}
-    ${dateFilters} and
-    ${queryFilter}
-    and transaction IN (
-      SELECT
-        transaction
-      FROM default.spans_experimental_starfish
-      WHERE ${queryFilter}
-      GROUP BY transaction
-      ORDER BY count() desc
-      LIMIT 5
-    )
-  group by transaction, interval
-  order by transaction, interval
-  `;
-
-  return useQuery({
-    enabled: !!transactionNames?.length,
-    queryKey: [
-      'EpmPerTransaction',
-      pageFilter.selection.datetime,
-      transactionNames.join(','),
-    ],
-    queryFn: () => fetch(`${HOST}/?query=${query}`).then(res => res.json()),
-    retry: false,
+): UseSpansQueryReturnType<QueryTransactionByTPMAndP75ReturnType> => {
+  const {
+    selection: {datetime},
+  } = usePageFilters();
+  return useSpansQuery<QueryTransactionByTPMAndP75ReturnType>({
+    eventView: EventView.fromSavedQuery({
+      name: '',
+      fields: ['transaction', 'count()', 'p75(transaction.duration)'],
+      yAxis: ['count()', 'p75(transaction.duration)'],
+      orderby: '-count',
+      query: `transaction:["${transactionNames.join('","')}"]`,
+      topEvents: '5',
+      start: datetime.start as string,
+      end: datetime.end as string,
+      range: datetime.period as string,
+      dataset: DiscoverDatasets.METRICS,
+      projects: [1],
+      version: 2,
+    }),
     initialData: [],
+    forceUseDiscover: true,
```

Comment:
small nit, but instead of `forceUseDiscover` we can probably just hit the discover hook directly instead as we always want discover and never experimental.